sdk/src/react/ui/modals/_internal/types/steps.ts

```diff
@@ -0,0 +1,132 @@
+export type StepStatus = 'idle' | 'pending' | 'complete' | 'error';
```

Comment:
success-error dual is more common, wdyt?

```suggestion
export type StepStatus = 'idle' | 'pending' | 'success' | 'error';
```

Reply:
Agreed, fixed